installbuilder/datafiles/Base_DSC.data

```diff
@@ -334,6 +300,9 @@ done
 
 #if BUILD_OMS == 1
 chown -R omsagent /etc/opt/omi/conf/${{SHORT_NAME}}
+
+# Set up the nxOMSAgent module
+/opt/microsoft/omsconfig/Scripts/InstallModule.py /opt/microsoft/omsconfig/module_packages/nxOMSAgent_1.0.zip 0
```

Comment:
Sounds good, I will split this one module up into different modules as requested.  AIso, I just investigated why it failed for you, Robbie.  It is due to the system not having "unzip" installed.  I'll need to make sure that's in the preinstall check for omsconfig now.